app/helpers/contact_helper.rb

```diff
@@ -0,0 +1,14 @@
+module ContactHelper
+  def contact_link(contact)
+    if contact.present?
+      text = safe_join([fa_icon('envelope'), ' ', contact.name])
+      link_to(text, contact_path(contact))
+    end
+  end
+
+  def contact_from_constant(constant:)
+    contact = Constant.find_by(name: constant).try(:value)
+    contact = Contact.find_by(id: contact)
```

Comment:
@jforberg 
I'll merge this and add to the  #221 to create a new column for a string that can be statically referenced and then we can skip one query.

This should perhaps be done for `Document` as well.